guides/hack/02-source-code-fundamentals/37-namespaces.md

```diff
@@ -52,11 +58,16 @@ namespace NS3\Sub1;
 ...                   // __NAMESPACE__ is "NS3\Sub1"
 }
 ```
+## Importing from other Namespaces
+With the `use` keyword, a namespace can import one or more member names into a scope, optionally giving them each an alias.
 
-In this case, the namespace extends from the opening brace to the closing brace.
+When importing many names, use `{ ... }`.
+
+```Hack
+use NS1\{C, I, T}; // instead of `NS1\C, NS1\I, NS1\T`
```

Comment:
We want to remove raw `use`, and only support `use <kind>`; for example:

```
use namespace HH\Lib\Vec;
use type SomeLib\SomeType;
use const SomeLib\SomeConst;
use function Facebook\Expect\expect;
```

`use Foo;` is essentially `use namespace Foo;` and `use class Foo;` (with some conflict resolution rules), for PHP compatibility which is no longer needed.

Reply:
Good to know. Fixed and will be on the lookout for other examples of ambiguity in the docs.